### PR TITLE
Develop user object

### LINKED ISSUE #2: Develop User Object
- Create user object
- Create user view
- Create user controller
- Create user unit tests

### LINKED ISSUE #8: sign-in page
- develop sign in controller
- develop sign-in page view 
- Unit test for the sign-in page

### PR DESCRIPTION
**Pull request by Favourolotu**
User interaction 
Unit tests
Sign in page integration

### CODE REVIEW

**Review by j-ctrl7 — COMMENTED** [review]
The changes look good

**Review by chase-scott — COMMENTED** [review]
I think all of our html files should follow a lower case naming convention with dashes

File: src/main/java/project/controllers/UserController.java [hunk 1]
```diff
@@ -39,8 +37,10 @@ public class UserController {
      */
     @GetMapping("/userDefaultPage")
     public String defaultDisplay(Model model) {
-        List<Book> books = (List<Book>) bookRepository.findAll();
-        model.addAttribute("books", books);
+        Iterable<Inventory> inventories = inventoryRepository.findAll();
+        // One shopping cart per user
+        shoppingCartRepository.save(new ShoppingCart());
```
Comment (chase-scott) [inline src/main/java/project/controllers/UserController.java:42]: Looks like you initialize a new shopping cart here but I don’t see where it is linked to the user

File: src/main/java/project/controllers/UserController.java [hunk 2]
```diff
@@ -74,50 +74,55 @@ public String searchForBooks(@RequestParam("bookName") String bookName, Model mo
      */
     @GetMapping("/viewShoppingCart")
     public String viewShoppingCart( Model model) {
-        // TODO get list of all books in the users shopping cart and display
 
-        //model.addAttribute("books", );
+        ShoppingCart shoppingCart = shoppingCartRepository.findShoppingCartById(1L) ;
```
Comment (chase-scott) [inline src/main/java/project/controllers/UserController.java:78]: I’m assuming this will be linked to the user in the next update

**Pull request merged**